tests/models/unets/test_models_unet_2d_condition.py

```diff
@@ -1119,6 +1119,24 @@ def test_load_attn_procs_raise_warning(self):
             lora_sample_1, lora_sample_2, atol=1e-4, rtol=1e-4
         ), "Loading from a saved checkpoint should produce identical results."
 
+    @require_peft_backend
+    def test_save_attn_procs_raise_warning(self):
+        init_dict, _ = self.prepare_init_args_and_inputs_for_common()
+        model = self.model_class(**init_dict)
+        model.to(torch_device)
+
+        unet_lora_config = get_unet_lora_config()
+        model.add_adapter(unet_lora_config)
+
+        assert check_if_lora_correctly_set(model), "Lora not correctly set in UNet."
+
+        with tempfile.TemporaryDirectory() as tmpdirname:
+            with self.assertWarns(FutureWarning) as warning:
```

Comment:
You could use `assertWarnsRegex` to check the message.

Reply:
We use `assertWarns` throughout `diffusers`. So, just kept that for consistency.